lib/memory/index.js

```diff
@@ -103,19 +103,22 @@ module.exports = function MemoryStore(data_set) {
   })
 
 
-  // _sort :: Sort -> [a] -> [a]
-  const _sort = R.curry((sort, list) => {
+  // _sortToFn :: Sort -> ComparatorFn
+  const _sortToFn = ([key, order]) => R.ifElse(
+    R.equals('desc')
```

Comment:
@jonajonlee if someone didn't spell 'asc' or 'desc' correctly, do we still want to sort?